src/views/open-api-docs-view-v2/index.tsx

```diff
@@ -10,14 +10,18 @@ import {
 	mobileMenuLevelMain,
 	mobileMenuLevelProduct,
 } from '@components/mobile-menu-levels/level-components'
-import { OpenApiV2SidebarContents } from './components/sidebar'
 import { SidebarHorizontalRule } from '@components/sidebar/components'
-import { SidebarResourceLinks } from './components/sidebar-resource-links'
-import { LandingContent } from './components/landing-content'
 import MobileMenuLevels from '@components/mobile-menu-levels'
-import OperationContent from './components/operation-content'
 import SidebarBackToLink from '@components/sidebar/components/sidebar-back-to-link'
 import BreadcrumbBar from '@components/breadcrumb-bar'
+import VersionSwitcher from '@components/version-switcher'
```

Comment:
Adds support for `versionMetadata` and `versionSwitcherProps`, and adds in the `OpenApiV2VersionAlert`, `NoIndexTagIfVersioned`, and `VersionSwitcher` components.